duckduckgo/chrome-zeroclickinfo
# use url as tosdr keys and update site tests

**Reviewer:**



## Description:

We're using just the site name as the keys in the tosdr.json file and using that to match against site domains. This means we could end up matching on incorrect sites. `instagram.somesite.com` would match on the `instagram` entry in tosdr.json.

To fix that I can switch to using the url provided by tosdr as the keys in tosdr.json.

I also updated our site tests so they work again and added tosdr site score tests.

## Steps to test this PR:

1. Run `make` to regenerate the tosdr.json data
2. Run the unit tests
3. Test some sites in tosdr.json. Check the extension console and you should see something for `site.score.tosdr`

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
